src/features/landing/components/ForWhoSection/Cards/Cards.stories.tsx

```
@@ -0,0 +1,18 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import Cards from './Cards';
+
+const meta: Meta<typeof Cards> = {
+  title: 'Components/ForWhoSection/Cards',
+  component: Cards,
+  tags: ['autodocs'],
+  parameters: {
+    layout: "", 
```

ESLint

[error] Strings must use singlequote.

(quotes)